### DISCUSSION TITLE
Black Screen When Opening Game

### DISCUSSION THREAD

**Discussion by DamazcoVolador** [mupen64plus-ae/mupen64plus-ae#1130] [General]
I cant play Because a dark screen doesnt let me see anything,but i can hear that im playing

**Comment by fzurita** [comment]
I'm what game? Have you tried switching emulation profiles?

**Comment by DamazcoVolador** [comment]
Nope,i will try, thanks

**Comment by DamazcoVolador** [comment]
Nope,it totally didnt work

**Comment by fzurita** [comment]
What game are you trying to play?

**Comment by DamazcoVolador** [comment]
A bit of rom hacks of super mario 64,and super mario 64

**Comment by fzurita** [comment]
Ok, do all the games give you a black screen or only specific ones?

**Comment by DamazcoVolador** [comment]
Every game i have

**Comment by fzurita** [comment]
Sounds like a compatibility issue with your device. What device do you have?

**Reply by mordanepic** [reply]
im having the same issue but it used to work im on a chromebook and it stopped working on my kindle fire randomly too any ideas as to why?

**Comment by DamazcoVolador** [comment]
Uhmmm,motorola E6S
